### PR TITLE
[zh-cn] update <code> html element

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Update translation of HTML `<code>` element.

### CODE REVIEW

File: files/zh-cn/web/html/element/code/index.md [hunk 1]
```diff
@@ -1,37 +1,97 @@
...
 slug: Web/HTML/Element/code
+l10n:
+  sourceCommit: fdd3ac5598c3ddceb71e59949b003936ae99f647
 ---
 
-## 摘要
+{{HTMLSidebar}}
 
-**HTML `<code>` 元素**呈现一段计算机代码。默认情况下，它以浏览器的默认等宽字体显示。
+[HTML](/zh-CN/docs/Web/HTML) **`<code>`** 元素显示其内容的样式，旨在表明文本是一段简短的计算机代码。默认情况下，内容文本使用 {{Glossary("user agent", "用户代理")}}默认的等宽字体显示。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/element/code/index.md:10]:
```suggestion
[HTML](/zh-CN/docs/Web/HTML) **`<code>`** 元素显示其内容的样式，旨在表明文本是一段简短的计算机代码。默认情况下，内容文本使用{{Glossary("user agent", "用户代理")}}默认的等宽字体显示。
```

File: files/zh-cn/web/html/element/code/index.md [hunk 2]
```diff
@@ -1,37 +1,101 @@
...
 slug: Web/HTML/Element/code
+l10n:
+  sourceCommit: fdd3ac5598c3ddceb71e59949b003936ae99f647
 ---
 
-## 摘要
+{{HTMLSidebar}}
 
-**HTML `<code>` 元素**呈现一段计算机代码。默认情况下，它以浏览器的默认等宽字体显示。
+[HTML](/zh-CN/docs/Web/HTML) **`<code>`** 元素显示其内容的样式，旨在表明文本是一段简短的计算机代码。默认情况下，内容文本使用{{Glossary("user agent", "用户代理")}}默认的等宽字体显示。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/code/index.md:10]:
```suggestion
[HTML](/zh-CN/docs/Web/HTML) **`<code>`** 元素为其显示的内容添加用以表明其中的文本是一段简短的计算机代码的样式。默认情况下，内容文本使用{{Glossary("user agent", "用户代理")}}默认的等宽字体显示。
```

File: files/zh-cn/web/html/element/code/index.md [hunk 3]
```diff
@@ -1,37 +1,101 @@
...
+        <a href="/zh-CN/docs/Web/HTML/Content_categories"
+          >内容分类</a
+        >
+      </th>
+      <td>
+        <a href="/zh-CN/docs/Web/HTML/Content_categories#流式内容"
+          >流式内容</a
+        >、<a href="/zh-CN/docs/Web/HTML/Content_categories#短语内容"
+          >短语内容</a
+        >、可感知内容
```
Comment (yin1999) [inline files/zh-cn/web/html/element/code/index.md:55]:
```suggestion
        >、可感知内容。
```

**Review by yin1999 — APPROVED** [review]
LGTM. Only two points for improvement

**Pull request merged**